### ISSUE TITLE
final database schema

### ISSUE THREAD

**Issue by joraco-dev** [joraco-dev/prometeo#5]
create a new db schema for the final product. must include all the tables and relations in order to don't miss any relevant figure

**Comment by joraco-dev** [comment]
first draft of the db schema

![image](https://user-images.githubusercontent.com/51737464/65379281-170eec00-dcc6-11e9-8e85-df8f6de690b4.png)

**Comment by krook** [comment]
@joraco-dev I believe I had seen a new version of the SQL database tables in Slack. Are those the latest?

**Comment by joraco-dev** [comment]
> @joraco-dev I believe I had seen a new version of the SQL database tables in Slack. Are those the latest?

yes, that will be the latest, but not yet implemented, as we're struggling with the security by row due to the need of securize all the data individually as the law requieres here in europe
